src/frequenz/sdk/actor/power_distributing/_component_managers/_pv_inverter_manager/_pv_inverter_manager.py

```diff
@@ -136,7 +136,8 @@ async def distribute_power(self, request: Request) -> None:
             if remaining_power > Power.zero() or is_close_to_zero(
```

Comment:
The `stop` is outdated in the comment above.

Reply:
I think it is still correct, because when we set 0 power to the remaining inverters, we're no longer allocating, but the comment could provide further details: https://github.com/frequenz-floss/frequenz-sdk-python/pull/960